src/Http/Requests/Conversations/GetConversationRequest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace RonasIT\Chat\Http\Requests\Conversations;
+
+use RonasIT\Chat\Contracts\Requests\GetConversationRequestContract;
+use RonasIT\Chat\Models\Conversation;
+use RonasIT\Chat\Services\ConversationService;
+use RonasIT\Support\BaseRequest;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+class GetConversationRequest extends BaseRequest implements GetConversationRequestContract
+{
+    protected ?Conversation $conversation;
+
+    public function rules(): array
+    {
+        return [
+            'with' => 'array',
+            'with.*' => "string|required",
+        ];
+    }
+
+    public function validateResolved()
+    {
+        $this->init();
+
+        parent::validateResolved();
+
+        $this->checkConversationExists();
+
+        $this->checkConversationOwnership();
+    }
+
+    protected function init()
+    {
+        $this->conversation = app(ConversationService::class)->find($this->route('id'));
+    }
+
+    protected function checkConversationExists()
+    {
+        if (empty($this->conversation)) {
+            throw new NotFoundHttpException(__('validation.exceptions.not_found', ['entity' => 'Conversation']));
```

Comment:
in other places you use

```suggestion
            throw new NotFoundHttpException(__('chat::validation.exceptions.not_found', ['entity' => 'Conversation']));
```